DataCollection/src/Infrastructure/Persistence/DataCollectionContext.cs

```diff
@@ -54,6 +55,24 @@ public class DataCollectionContext : DbContext, ILocationPersistence, IWorkLogPe
         }
         #endregion
 
+        #region Specialized Queries for IBirdSpeciesPersistence
+
+        public IEnumerable<BirdSpecies> GetBirdSpeciesBySurveyTypeId(int surveyTypeId)
+        {
+            var returnVal = new List<BirdSpecies>();
+            var surveyType = SurveyTypes.Include(survey => survey.BirdSpecies).FirstOrDefault(surveyItem => surveyItem.Id == surveyTypeId);
+
+            if (surveyType != null)
+            {
+                if (surveyType.BirdSpecies != null)
+                {
+                    returnVal = surveyType.BirdSpecies.ToList();
+                }
+            }
+            return returnVal;
```

Comment:
Can you move this to the manager class? I'm trying to leave the `DataCollectionContext` class as simple as possible, representing the data model but not representing queries. I think you could just move this query right over to the manager class. You might need a new interface for `DataCollectionContext`, like `ILocationPersistence` for example.